guide/the-3d-window.md

```diff
@@ -99,9 +98,9 @@ key and right mouse button together while dragging the mouse. Linux users should
 also hold down the Control key (Ctrl) together with the Alt key. Also, Mac OS X
 users with a one-button mouse should hold down the Control key (Ctrl) to emulate
 the right mouse button. This way your are drawing a 3D-vector with origin the
-center of mass and whose end is located in the plane parallel to the view which
+center of mass and whose end is located on the plane parallel to the view which
 passes through this center. The object is prompted to turn around the vector
-direction, the intensity of the applied torque being directly proportional to
+direction, the intensity of the applied torque directly proportional to
```

Comment:
Why did you remove "being"? I think it's useful. Maybe we could simplify and replace it with "is".